### DISCUSSION TITLE
Different ratings source based on genre?

### DISCUSSION THREAD

**Discussion by alexkahler** [meisnate12/Plex-Meta-Manager#1893] [Q&A] [answered]
Hi!

Is there any way to use MAL ratings for Anime and TMDb ratings for regular TV Shows in a unified library (both anime and TV shows are in 1 library) and then applying overlays?

**Comment by YozoraXCII** [comment, marked as answer]
This is not possible, no

**Reply by alexkahler** [reply]
Thanks. I ended up splitting TV shows and Anime into 2 different libraries instead 👍
